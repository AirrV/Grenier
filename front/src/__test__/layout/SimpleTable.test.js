import React from 'react';
import renderer from "react-test-renderer";
import SimpleTable from "../../components/layout/SimpleTable.js"

it("Renders the welcome page", () => {
  const tree = renderer
    .create(<SimpleTable />)
    .toJSON();
  expect(tree).toMatchSnapshot();
})