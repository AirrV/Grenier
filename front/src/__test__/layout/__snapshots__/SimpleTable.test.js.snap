// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the welcome page 1`] = `
<div
  className="MuiPaper-root-3 MuiPaper-elevation2-7 MuiPaper-rounded-4 SimpleTable-root-1"
>
  <table
    className="MuiTable-root-30 SimpleTable-table-2"
  >
    <thead
      className="MuiTableHead-root-31"
    >
      <tr
        className="MuiTableRow-root-32 MuiTableRow-head-35"
      >
        <th
          aria-sort={null}
          className="MuiTableCell-root-37 MuiTableCell-head-38"
          scope="col"
        />
        <th
          align="right"
          aria-sort={null}
          className="MuiTableCell-root-37 MuiTableCell-head-38"
          scope="col"
        >
          Quantité / Poids
        </th>
        <th
          align="right"
          aria-sort={null}
          className="MuiTableCell-root-37 MuiTableCell-head-38"
          scope="col"
        >
          Temps
        </th>
        <th
          align="right"
          aria-sort={null}
          className="MuiTableCell-root-37 MuiTableCell-head-38"
          scope="col"
        >
          Prix
        </th>
      </tr>
    </thead>
    <tbody
      className="MuiTableBody-root-45"
    >
      <tr
        className="MuiTableRow-root-32"
      >
        <th
          aria-sort={null}
          className="MuiTableCell-root-37 MuiTableCell-body-39"
          scope="row"
        >
          Saucisse sèche
        </th>
        <td
          align="right"
          aria-sort={null}
          className="MuiTableCell-root-37 MuiTableCell-body-39"
        >
          300 g
        </td>
        <td
          align="right"
          aria-sort={null}
          className="MuiTableCell-root-37 MuiTableCell-body-39"
        >
          2 jours
        </td>
        <td
          align="right"
          aria-sort={null}
          className="MuiTableCell-root-37 MuiTableCell-body-39"
        >
          5€
        </td>
      </tr>
      <tr
        className="MuiTableRow-root-32"
      >
        <th
          aria-sort={null}
          className="MuiTableCell-root-37 MuiTableCell-body-39"
          scope="row"
        >
          Pâtes
        </th>
        <td
          align="right"
          aria-sort={null}
          className="MuiTableCell-root-37 MuiTableCell-body-39"
        >
          1 kg
        </td>
        <td
          align="right"
          aria-sort={null}
          className="MuiTableCell-root-37 MuiTableCell-body-39"
        >
          1 jour
        </td>
        <td
          align="right"
          aria-sort={null}
          className="MuiTableCell-root-37 MuiTableCell-body-39"
        >
          5€
        </td>
      </tr>
      <tr
        className="MuiTableRow-root-32"
      >
        <th
          aria-sort={null}
          className="MuiTableCell-root-37 MuiTableCell-body-39"
          scope="row"
        >
          Cochon entier
        </th>
        <td
          align="right"
          aria-sort={null}
          className="MuiTableCell-root-37 MuiTableCell-body-39"
        >
          320kg
        </td>
        <td
          align="right"
          aria-sort={null}
          className="MuiTableCell-root-37 MuiTableCell-body-39"
        >
          6 mois
        </td>
        <td
          align="right"
          aria-sort={null}
          className="MuiTableCell-root-37 MuiTableCell-body-39"
        >
          5€
        </td>
      </tr>
      <tr
        className="MuiTableRow-root-32"
      >
        <th
          aria-sort={null}
          className="MuiTableCell-root-37 MuiTableCell-body-39"
          scope="row"
        >
          Courgettes
        </th>
        <td
          align="right"
          aria-sort={null}
          className="MuiTableCell-root-37 MuiTableCell-body-39"
        >
          500 g
        </td>
        <td
          align="right"
          aria-sort={null}
          className="MuiTableCell-root-37 MuiTableCell-body-39"
        >
          2 jours
        </td>
        <td
          align="right"
          aria-sort={null}
          className="MuiTableCell-root-37 MuiTableCell-body-39"
        >
          5€
        </td>
      </tr>
      <tr
        className="MuiTableRow-root-32"
      >
        <th
          aria-sort={null}
          className="MuiTableCell-root-37 MuiTableCell-body-39"
          scope="row"
        >
          Œufs
        </th>
        <td
          align="right"
          aria-sort={null}
          className="MuiTableCell-root-37 MuiTableCell-body-39"
        >
          6
        </td>
        <td
          align="right"
          aria-sort={null}
          className="MuiTableCell-root-37 MuiTableCell-body-39"
        >
          1 semaine
        </td>
        <td
          align="right"
          aria-sort={null}
          className="MuiTableCell-root-37 MuiTableCell-body-39"
        >
          5€
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
